### PR TITLE
Sarthak Sharma 2019A4PS0449P

### PR DESCRIPTION
**Pull request by SARTHAKKRSHARMA**

### CODE REVIEW

**Comment by SARTHAKKRSHARMA** [comment]
It was fun working on this assignment.

**Review by 7vik — CHANGES_REQUESTED** [review]
Great Job! Have you created your own directory as suggested in the instructions? @SARTHAKKRSHARMA

**Comment by twitu** [comment]
@SARTHAKKRSHARMA please move all your files to a single directory, as mentioned in the instructions. I can merge your PR after that.

File: Problem3.hs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+elementAt :: [a] -> Int -> a
+elementAt xs n = xs !! (n - 1)
+
```
Comment (twitu) [inline Problem3.hs]: Try to come up with one more solution to this problem, as a hint you can create a function called,
`elementAt :: [a] -> Int -> a` that actually calls another function that has a slightly different type signature namely,
`actualElementAt :: ([a], Int) -> Int -> a`.

Comment (twitu) [reply Problem3.hs]: On the other hand you could use an approach similar to the `myLength` function.

File: Problem6.hs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import Data.Char
+mytoLower :: Char -> Char
+mytoLower ch
+    | ord 'A' <= ord ch && ord 'Z' >= ord ch  = chr (ord ch + ord 'a' - ord 'A')
```
Comment (twitu) [inline Problem6.hs]: Look for a function with type signature `Char -> Char` in https://hoogle.haskell.org/, it might give you some ideas ;)

**Comment by twitu** [comment]
The suggested improvements in code are something you can do in time. **However I need you to move your files to a single folder to merge the request.**

**Comment by SARTHAKKRSHARMA** [comment]
I have tried to move them under one directory but I'm not sure I have done
it correctly.

**Comment by twitu** [comment]
Cmon, friend at least think of what you are doing? Change `Folder1` to your name so I can at least identify your code later.

Take a look at this PR https://github.com/p-paf/99problems/pull/5, which has been merged.

**Review by twitu — APPROVED** [review]

**Comment by twitu** [comment]
You still have a few changes to make namely,
1. adding better implementations
2. naming all the files starting with capital P `Problem..` as per convention.

You can submit that as a separate PR.

**Comment by SARTHAKKRSHARMA** [comment]
How can I make better implementations?

**Comment by twitu** [comment]
I mean add more implementations as mentioned in the previous comments.

**Comment by SARTHAKKRSHARMA** [comment]
Surely I will.

**Pull request merged**